integreat_cms/cms/views/push_notifications/push_notification_form_view.py

```diff
@@ -69,32 +70,57 @@ def get(self, request, *args, **kwargs):
             kwargs.get("language_slug"), only_active=True
         )
 
-        push_notification = PushNotification.objects.filter(
+        push_notification = region.push_notifications.filter(
             id=kwargs.get("push_notification_id")
         ).first()
```

Comment:
Ah, true. Wouldn't it be better though to redirect to the proper *New Push Notification* URL? This is similar to my question whether it would make sense to switch to randomised IDs, the rationale would be not to disclose to the client whether they hit a valid ID or not, only that they didn't hit the ID of an object they are allowed to access